neuralforecast/_modidx.py

```diff
@@ -500,60 +500,7 @@
                                                                                                          'neuralforecast/losses/pytorch.py'),
                                                'neuralforecast.losses.pytorch.weighted_average': ( 'losses.pytorch.html#weighted_average',
                                                                                                    'neuralforecast/losses/pytorch.py')},
-            'neuralforecast.models.autoformer': { 'neuralforecast.models.autoformer.AutoCorrelation': ( 'models.autoformer.html#autocorrelation',
```

Comment:
can you please install "nbdev==2.3.25"? the latest version removes these symbols